src/apps/inventory/models.py

```diff
@@ -1,14 +1,24 @@
 from django.conf import settings
 from django.db import models
+from django.utils.text import slugify
 
 
 class Category(models.Model):
     name = models.CharField(max_length=255)
     description = models.TextField(blank=True)
+    slug = models.SlugField(default=str(name))
```

Comment:
Does it make sense? Feels like default arg should be omitted

Reply:
fixed